tsconfig.json

```diff
@@ -21,5 +21,5 @@
     "noEmit": true,
     "jsx": "react-jsx"
   },
-  "include": ["src"]
+  "include": ["src", "src/components/QrScan/qr-scanner.min.js"]
```

Comment:
I don't think this change is needed since that file would already be included by the `src` path included before it.